docs/API_SPECS_DEVICE_MODEL_SCRIPTS.md

```diff
@@ -0,0 +1,183 @@
+      "Path": "Storage",
+      "$metadata": {
+        "$type": "SimulationScript;1",
+        "$uri": "/v1/simulationscripts/371e7ef7-9d99-4beb-a934-f3914faa02a3",
+        "$created": "2018-07-31T22:40:48+00:00",
+        "$modified": "2018-07-31T22:40:48+00:00"
+      }
+    },
+    {
+      "ETag": "\"01005386-0000-0000-0000-5b60e5700000\"",
+      "Id": "f1943029-a1af-4952-bdf6-55a8311e0fab",
+      "Type": "javascript",
+      "Name": "IncreasePressure-method.js",
+      "Content": "// Copyright (c) Microsoft. All rights reserved.\n\n/*global log*/\n/*global updateState*/\n/*global sleep*/\n/*jslint node: true*/\n\n\"use strict\";\n\n/**\n * Entry point function called by the simulation engine.\n *\n * @param context        The context contains current time, device model and id, not used\n * @param previousState  The device state since the last iteration, not used\n * @param previousProperties  The device properties since the last iteration\n */\n/*jslint unparam: true*/\nfunction main(context, previousState, previousProperties) {\n\n    log(\"Executing 'IncreasePressure' JavaScript method simulation (5 seconds).\");\n\n    // Pause the simulation and change the simulation mode so that the\n    // pressure will fluctuate at ~250 when it resumes\n    var state = {\n        simulation_state: \"high_pressure\",\n        CalculateRandomizedTelemetry: false\n    };\n    updateState(state);\n\n    // Increase\n    state.pressure = 170;\n    updateState(state);\n    log(\"Pressure increased to \" + state.pressure);\n    sleep(1000);\n\n    // Increase\n    state.pressure = 190;\n    updateState(state);\n    log(\"Pressure increased to \" + state.pressure);\n    sleep(1000);\n\n    // Increase\n    state.pressure = 210;\n    updateState(state);\n    log(\"Pressure increased to \" + state.pressure);\n    sleep(1000);\n\n    // Increase\n    state.pressure = 230;\n    updateState(state);\n    log(\"Pressure increased to \" + state.pressure);\n    sleep(1000);\n\n    // Increase\n    state.pressure = 250;\n    updateState(state);\n    log(\"Pressure increased to \" + state.pressure);\n    sleep(1000);\n\n    // Resume the simulation\n    state.CalculateRandomizedTelemetry = true;\n    updateState(state);\n\n    log(\"'IncreasePressure' JavaScript method simulation completed.\");\n}\n",
+      "Path": "Storage",
+      "$metadata": {
+        "$type": "DeviceModelScript;1",
+        "$uri": "/v1/devicemodelscripts/f1943029-a1af-4952-bdf6-55a8311e0fab",
+        "$created": "2018-07-31T22:40:49+00:00",
+        "$modified": "2018-07-31T22:40:49+00:00"
+      }
+    }
+  ],
+  "$metadata": {
+    "$type": "DeviceModelScriptList;1",
+    "$uri": "/v1/devicemodelscripts"
+  }
+}
+```
+
+### Get a device model script by id
+
+Request:
+```
+GET /v1/devicemodelscripts/${id}
```

Comment:
the use of `$` is not consistent with line 178 below

Reply:
Done.